
package com.nortel.soa.oi.cct.connectionservice;

import javax.xml.ws.WebFault;


/**
 * This class was generated by Apache CXF 2.6.1
 * 2014-01-31T11:02:30.265+03:00
 * Generated source version: 2.6.1
 */

@WebFault(name = "AcceptException", targetNamespace = "http://www.nortel.com/soa/oi/cct/faults")
public class AcceptException extends Exception {
    
    private org.oasis_open.docs.wsrf._2004._06.wsrf_ws_basefaults_1_2_draft_01.ExceptionInfo acceptException;

    public AcceptException() {
        super();
    }
    
    public AcceptException(String message) {
        super(message);
    }
    
    public AcceptException(String message, Throwable cause) {
        super(message, cause);
    }

    public AcceptException(String message, org.oasis_open.docs.wsrf._2004._06.wsrf_ws_basefaults_1_2_draft_01.ExceptionInfo acceptException) {
        super(message);
        this.acceptException = acceptException;
    }

    public AcceptException(String message, org.oasis_open.docs.wsrf._2004._06.wsrf_ws_basefaults_1_2_draft_01.ExceptionInfo acceptException, Throwable cause) {
        super(message, cause);
        this.acceptException = acceptException;
    }

    public org.oasis_open.docs.wsrf._2004._06.wsrf_ws_basefaults_1_2_draft_01.ExceptionInfo getFaultInfo() {
        return this.acceptException;
    }
}
